<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView_Spielfeld">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_xpos">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>550</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>XPOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ypos">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>610</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>YPOS</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_xpos">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>560</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_ypos">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>610</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
